assets/javascripts/modules/checkout/validatePayment.js

```diff
@@ -1,3 +1,4 @@
+/* global Raven */
```

Comment:
Any reason why this has to be a global rather than passing it as an explicit dependency via require.js?